### ISSUE TITLE
ipset: Set cannot be destroyed when running alongside keepalived-vip on the same host

### ISSUE THREAD

**Issue by stephan2012** [weaveworks/weave#2751]
When [keepalived-vip](https://github.com/kubernetes/contrib/tree/master/keepalived-vip) and Weave run on the same host, the weave-npc container constantly crashes:

```
# docker logs -f a95eb6b0429ea321c5494f8321413b082c5f3593dbbcea4b1dda1e5a7f140c0b 
time="2017-01-25T10:17:01Z" level=info msg="Starting Weaveworks NPC 1.8.2" 
time="2017-01-25T10:17:01Z" level=info msg="Serving /metrics on :6781" 
Wed Jan 25 10:17:01 2017 <5> ulogd.c:843 building new pluginstance stack: 'log1:NFLOG,base1:BASE,pcap1:PCAP'
time="2017-01-25T10:17:02Z" level=fatal msg="ipset [destroy] failed: ipset v6.29: Set cannot be destroyed: it is in use by a kernel component\n: exit status 1" 
```

Apparently Weave does not expect ipsets created by others. Only solution to bring Weave back working at the moment is to remove keepalived-vip and reboot the host.

Any ideas on how to make Weave co-exist with keepalived-vip on the same host?

Environment: Kubernetes 1.5, CentOS 7.3

**Comment by awh** [comment]
> Any ideas on how to make Weave co-exist with keepalived-vip on the same host?

Yes, fix this: https://github.com/weaveworks/weave/blob/master/prog/weave-npc/main.go#L75

**Comment by marccarre** [comment]
Closed by #2846 (for some reason it didn't get automatically closed by the PR).
